src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToSubmitHandler.java

```diff
@@ -140,6 +142,10 @@ private SscsCaseData updateCaseForDirectionType(CaseDetails<SscsCaseData> caseDe
             caseData.setInterlocReviewState(AWAITING_ADMIN_ACTION.getId());
             updateDwpRegionalCentre(caseData);
 
+            //Set digital flag on legacy cases
+            if (caseData.getCreatedInGapsFrom() == null || VALID_APPEAL.getId().equalsIgnoreCase(caseData.getCreatedInGapsFrom())) {
+                caseData.setCreatedInGapsFrom(READY_TO_LIST.getId());
```

Comment:
could we avoid null value

Reply:
Not sure what you mean by 'avoid null value' ?